samples/subsys/display/lvgl/README.rst

```diff
@@ -2,16 +2,27 @@
    :name: LVGL basic sample
    :relevant-api: display_interface
 
-   Display "Hello World" and a dynamic counter using LVGL.
+   Demonstrate LVGL input devices.
```

Comment:
I wouldn't put the emphasis on input devices only, as this is still supposed to be a "Hello World" sample (or is it? maybe not... and maybe we now have too many #ifdefs and overhead to the point where an "actual", simple, Hello World would be needed?)

```suggestion
   Display a "Hello World" and react to user input using LVGL.
```